src/wp-includes/blocks.php

```diff
@@ -794,16 +795,22 @@ function get_comment_delimited_block_content( $block_name, $block_attributes, $b
  * instead preserve the markup as parsed.
  *
  * @since 5.3.1
+ * @since 6.4.0 The `$callback` parameter was added.
  *
  * @param array $block A representative array of a single parsed block object. See WP_Block_Parser_Block.
+ * @param string $callback Optional. Callback to run on the block before serialization.
```

Comment:
Per the [PHP Inline Documentation Standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods):
- Slight alignment adjustments are needed for the parameters.
- The default value for an `Optional` parameter should be specified.

:1234: Applies to `serialize_blocks()` as well.

```suggestion
 * @param array  $block    A representative array of a single parsed block object. See WP_Block_Parser_Block.
 * @param string $callback Optional. Callback to run on the block before serialization. Default null.
```

Reply:
> 🔢 Applies to serialize_blocks() as well.

Addressed in https://github.com/WordPress/wordpress-develop/pull/5158/commits/5cd2ded3b6d1d057fa65608146ee81a74f785ea6.